### PR TITLE
NAS-103464 Fix a boot-env tooltip

### PR DESCRIPTION
**Pull request by dkmullen**

### CODE REVIEW

File: src/app/helptext/system/bootenv.ts [hunk 1]
```diff
@@ -13,19 +16,15 @@ export const helptext_system_bootenv = {
 
   clone_name_placeholder: T("Name"),
   clone_name_tooltip: T(
-    "Enter a name for the clone of this boot environment.\
- Alphanumeric characters, dashes (*-*), underscores (*_*),\
- and periods (*.*) are allowed.)"
+    `Enter a name for the clone of this boot environment. ${allowedCharacters}`
```
Comment (erinix) [inline src/app/helptext/system/bootenv.ts]: Unfortunately I don't think the translation extractor is smart enough to handle variables

**Comment by erinix** [comment]
please review @freenas/docs-team

File: src/app/helptext/system/bootenv.ts (hunk 1 above)
Comment (erinix) [reply src/app/helptext/system/bootenv.ts]: it won't work for the others either ;)

Comment (erinix) [reply src/app/helptext/system/bootenv.ts]: It still won't work with the way you changed it, there is no way to engineer around this one unfortunately :-/

File: src/app/helptext/system/bootenv.ts [hunk 2]
```diff
@@ -10,22 +10,20 @@ export const helptext_system_bootenv = {
...
   ),
+  allowedCharacters : T('Alphanumeric characters, dashes (-), underscores (_), \
+   and periods (.) are allowed.'),
 
   clone_name_placeholder: T("Name"),
   clone_name_tooltip: T(
-    "Enter a name for the clone of this boot environment.\
- Alphanumeric characters, dashes (*-*), underscores (*_*),\
- and periods (*.*) are allowed.)"
+    `Enter a name for the clone of this boot environment.`
```
Comment (Mrt134) [inline src/app/helptext/system/bootenv.ts]:
```suggestion
    `Name of the new cloned boot environment.`
```

File: src/app/helptext/system/bootenv.ts [hunk 3]
```diff
@@ -10,22 +10,20 @@ export const helptext_system_bootenv = {
...
   ),
   
   clone_source_placeholder: T("Source"),
   clone_source_tooltip: T("This is the boot environment to be cloned."),
```
Comment (Mrt134) [inline src/app/helptext/system/bootenv.ts]:
```suggestion
  clone_source_tooltip: T("Boot environment to be cloned."),
```

File: src/app/helptext/system/bootenv.ts [hunk 4]
```diff
@@ -10,22 +10,20 @@ export const helptext_system_bootenv = {
...
 
   create_name_placeholder: T("Name"),
   create_name_tooltip: T(
-    "Enter the name of the boot entry.\
- Alphanumeric characters, dashes (*-*), underscores (*_*),\
- and periods (*.*) are allowed.)"
+    `Enter the name of the boot entry.`
```
Comment (Mrt134) [inline src/app/helptext/system/bootenv.ts]:
```suggestion
    `Boot environment name.`
```

File: src/app/helptext/system/bootenv.ts [hunk 5]
```diff
@@ -35,9 +33,7 @@ export const helptext_system_bootenv = {
 
   rename_name_placeholder: T("Name"),
   rename_name_tooltip: T(
```
Comment (Mrt134) [inline src/app/helptext/system/bootenv.ts]: This helptext can be discarded in favor of reusing the "create_name_tooltip" - the help text should be identical.

File: src/app/helptext/system/bootenv.ts (hunk 1 above)
Comment (dkmullen) [reply src/app/helptext/system/bootenv.ts]: I don't think I understand. All of our tooltips are variables in the component pointing to helptext. This one is two variables.

**Pull request merged**